doc/api-examples/enable-unit.py

```diff
@@ -1,4 +1,5 @@
 #!/usr/bin/python3
+# SPDX-License-Identifier: GPL-2.0-or-later
```

Comment:
I think we should make the example code be of a more relax license - MIT or BSD or even CC0 could be considered

Reply:
According to our license agreement those files are already published under GPL-2.0-or-later: https://github.com/containers/bluechi/tree/main/LICENSES#main-license

So this change just make the license more visible and easier to check.

If we would need to change the license of those files, then we would probably need a consent from all person contributed to those files, right?

Reply:
fair, although, I would recommend we make these code snippet available under a different license

Reply:
Yes they should be dropped down to Apache or something like it, so users could modify the script and not have to redistribute it.

Reply:
Fixed by https://github.com/containers/bluechi/pull/476 and https://github.com/containers/bluechi/pull/478